src/sentry/monitors/validators.py

```diff
@@ -410,3 +410,17 @@ def validate_slugs(self, value):
         ).count() != len(value):
             raise ValidationError("Not all slugs are valid for this organization.")
         return value
+
+
+class MonitorBulkEditValidator(MonitorValidator):
+    ids = serializers.ListField(
+        child=serializers.UUIDField(format="hex"),
+        required=True,
+    )
+
+    def validate_ids(self, value):
+        if Monitor.objects.filter(
+            guid__in=value,
+        ).count() != len(value):
+            raise ValidationError("Not all ids are valid for this organization.")
```

Comment:
This doesn't look like it's actually doing an organization check like the one in the legacy validator?

Reply:
ooh good catch, yea we should make sure the guids they specify are all from their own organization